airflow/operators/python_operator.py

```diff
@@ -29,6 +30,9 @@
 from textwrap import dedent
 
 
+AIRFLOW_HOME_VAR = 'AIRFLOW_HOME'
```

Comment:
Shouldn't this be in the conf? and imported at the same time? If this changes for any reason, people won't know to look for it inside here

Reply:
@artwr the `BashOperator` does something similar.

Reply:
the env var should be de-dupped at least.

Reply:
@aoen it seems that the subprocess command doesn't support prepending environment variables in the same manner of the [BashOperator](https://github.com/apache/incubator-airflow/blob/master/airflow/operators/bash_operator.py#L84), also it's not apparent what shell is being run. From a Google search the approach I've taken here with regards to augmenting the environment seems to be consistent with others.